src/frontend/src/pages/FinancePage/FinanceComponents/GenerateReceiptsModal.tsx

```diff
@@ -104,9 +113,7 @@ const GenerateReceiptsModal = ({
                   onClose={() => setStartDatePickerOpen(false)}
                   onOpen={() => setStartDatePickerOpen(true)}
                   onChange={(newValue) => {
-                    const newDate = newValue ?? new Date();
-                    setStartDate(newDate);
-                    onChange(newDate);
+                    onChange(newValue ?? new Date());
```

Comment:
I think you might have deleted this when fixing the merge conflict and went back to the original